htmd/builder/amber.py

```diff
@@ -406,26 +431,40 @@ def build(mol, ff=None, topo=None, param=None, prefix='structure', outdir='./bui
 
     molbuilt = None
     if execute:
-        # Source paths of extra dirs (our dirs, not amber default)
-        htmdamberdir = os.path.abspath(os.path.join(home(), 'builder', 'amberfiles'))
-        sourcepaths = [htmdamberdir]
-        sourcepaths += [os.path.join(htmdamberdir, os.path.dirname(f))
-                        for f in ff if os.path.isfile(os.path.join(htmdamberdir, f))]
-        extrasource = []
-        for p in sourcepaths:
-            extrasource.append('-I')
-            extrasource.append('{}'.format(p))
+        if not teleapimports:
+            teleapimports = []
+            # Source default Amber (i.e. the same paths tleap imports)
+            amberhome = defaultAmberHome(teleap=teleap)
+            teleapimports.append(os.path.join(amberhome, 'dat', 'leap', 'prep'))
+            teleapimports.append(os.path.join(amberhome, 'dat', 'leap', 'lib'))
+            teleapimports.append(os.path.join(amberhome, 'dat', 'leap', 'parm'))
+            teleapimports.append(os.path.join(amberhome, 'dat', 'leap', 'cmd'))
```

Comment:
Why not do these like in charmm.build where you have a `defaultFF` function that returns them?

Reply:
In Amber we also have the `defaultFF` functions. But they return files, while these are search paths. To be honest, I wanted to unify them and replace their use in `listFiles` and `_locateFile`, but I thought that it was already too much refactoring. But if you agree with it, I could create some private functions that return those default paths, perhaps in a dictionary.

Reply:
Yes, maybe do that if it's not a bother. It's a bit ugly like this.

Reply:
It's not, I wanted to do it, but maybe too much refactoring. Let me check it.

Reply:
I think I was able to do what you asked in a very simple and clean way. Let me know if you like it.